CoreFoundation/RunLoop.subproj/CFSocket.c

```diff
@@ -2625,7 +2625,7 @@ static CFSocketRef _CFSocketCreateWithNative(CFAllocatorRef allocator, CFSocketN
 #endif
         pthread_create(&tid, &attr, __CFSocketManager, 0);
         pthread_attr_destroy(&attr);
-//warning CF: we dont actually know that a pthread_t is the same size as void *
+//warning CF: we dont actually know that a _CFThreadRef is the same size as void *
```

Comment:
Could add (C11 has this now):

    _Static_assert(sizeof(void* ) == sizeof(CFThreadRef), "sizeof(void* ) != sizeof(CFThreadRef)");

Reply:
Is CF built assuming C11 or does it attempt to compatible with C99?  If so, I'm happy to add the assertion.

Reply:
Seems unlikely that anyone would attempt to compile this without a C11-compatible compiler at this point, but you could guard it with an ifdef, or check with @parkera, who should know for sure.